src/ExpressionEvaluator/Core/Test/ResultProvider/Debugger/Engine/DkmClrValue.cs

```diff
@@ -156,7 +151,7 @@ public bool HasUnderlyingString(DkmInspectionContext inspectionContext)
                 throw new ArgumentNullException(nameof(inspectionContext));
             }
 
-            return _formatter.HasUnderlyingString(this, inspectionContext);
+            return inspectionContext.InspectionSession.InvokeFormatter(MethodId.HasUnderlyingString, f => f.HasUnderlyingString(this, inspectionContext));
```

Comment:
Wow...I've never seen this before...  Are we going to add this method to the InspectionContext eventually?